cgc/Mapping_file_lucky_50.txt

```diff
@@ -0,0 +1 @@
s32	AAAAAAAAAA	TTTTTTTTTT	TCGA-06-0128-01A-01D-0572-09_whole.bam	Primary Tumor	WGS	FF6B6D81-68C6-4583-8A25-D914428BB23D
s33	AAAAAAAAAA	TTTTTTTTTT	TCGA-AB-2920-03B-01T-0760-13_rnaseq.bam	Primary Blood Derived Cancer - Peripheral Blood	RNA-Seq	FF6688AF-BFB8-40E0-8087-F7C38C35FE4F
s34	AAAAAAAAAA	TTTTTTTTTT	TCGA-A5-A0GX-10A-01D-A043_120327_SN590_0148_BC0LM8ACXX_s_8_rg.sorted.bam	Blood Derived Normal	WGS	FF637ADE-4B5A-4A0A-9712-4FF2FC5E5B9D
s35	AAAAAAAAAA	TTTTTTTTTT	G23078.TCGA_CR_6487_10A_01D_2268_08.1.bam	Blood Derived Normal	WGS	FF63290D-0644-4387-99BB-697150A7FB6B
s36	AAAAAAAAAA	TTTTTTTTTT	TCGA-38-4631-11A-01D-1751_120305_SN1120_0118_AD0L4HACXX_s_6_rg.sorted.bam	Solid Tissue Normal	WGS	FF565F35-43C7-4694-B7D0-70F440732B1C
s37	AAAAAAAAAA	TTTTTTTTTT	TCGA-AB-2887-03A-01T-0735-13_rnaseq.bam	Primary Blood Derived Cancer - Peripheral Blood	RNA-Seq	FF4FF433-F4F9-48E3-A02E-4C39BEE265CB
s38	AAAAAAAAAA	TTTTTTTTTT	UNCID_1541326.5cb5c8ef-740d-4f9d-b081-107a3e4390f6.sorted_genome_alignments.bam	Primary Tumor	RNA-Seq	FF4195CC-E15C-4E9D-9CAA-F74D7F0681D9
s39	AAAAAAAAAA	TTTTTTTTTT	G17506.TCGA-27-1835-01A-01R-1850-01.2.bam	Primary Tumor	RNA-Seq	FF4195CC-E15C-4E9D-9CAA-F74D7F0681D9
s40	AAAAAAAAAA	TTTTTTTTTT	UNCID_2306132.66159afa-029a-49f4-a260-f9a405444140.sorted_genome_alignments.bam	Primary Tumor	RNA-Seq	FF3E53E7-90BE-48CE-B729-9A30FC7FCE92
s41	AAAAAAAAAA	TTTTTTTTTT	UNCID_1201076.3a02dae1-49a3-430d-9075-2d9d4bad6593.sorted_genome_alignments.bam	Primary Tumor	RNA-Seq	FF39C988-3D62-4BBD-9453-82ADE6E1D44C
s42	AAAAAAAAAA	TTTTTTTTTT	UNCID_374597.TCGA-A3-3320-01A-02R-1325-07.110421...2ABXX.8.trimmed.annotated.translated_to_genomic.bam	Primary Tumor	RNA-Seq	FF39C988-3D62-4BBD-9453-82ADE6E1D44C
s43	AAAAAAAAAA	TTTTTTTTTT	UNCID_2152931.bf4dc301-6b99-46c3-bdc4-d85905688e05.sorted_genome_alignments.bam	Primary Tumor	RNA-Seq	FF35CA1E-064F-4C89-9BBE-01D26C5C57BF
s44	AAAAAAAAAA	TTTTTTTTTT	UNCID_2362245.bcaa4853-de3c-479c-83ac-008181336cba.sorted_genome_alignments.bam	Primary Tumor	RNA-Seq	FF304E13-8DDE-4EB6-815F-A81EC3C4A570
s45	AAAAAAAAAA	TTTTTTTTTT	G47088.TCGA-77-6843-10A-01D-1945-08.1.bam	Blood Derived Normal	WGS	FF240243-CF04-4CA9-B829-3BBD459FF8F5
s46	AAAAAAAAAA	TTTTTTTTTT	UNCID_2542925.b9b42bfa-2222-4d21-a163-02f767c4db38.sorted_genome_alignments.bam	Primary Tumor	RNA-Seq	FF206E85-427C-4BA2-8ACB-A1D603E9C02F
s47	AAAAAAAAAA	TTTTTTTTTT	UNCID_1781098.7dab0105-975d-459f-97d3-f1462afddd4d.sorted_genome_alignments.bam	Primary Tumor	RNA-Seq	FF1F5513-8463-4B1E-B254-72D9E0863D35
s48	AAAAAAAAAA	TTTTTTTTTT	TCGA-G2-A2EK-01A-22D-A18D_120913_SN208_0429_AC187FACXX_s_3_rg.sorted.bam	Primary Tumor	WGS	FF1691F2-DF11-4068-8D8D-A778F384EC60
s49	AAAAAAAAAA	TTTTTTTTTT	UNCID_1210996.bc993392-5f2f-4cb8-8d27-26f3df2b3583.sorted_genome_alignments.bam	Primary Tumor	RNA-Seq	FF1691F2-DF11-4068-8D8D-A778F384EC60
s50	AAAAAAAAAA	TTTTTTTTTT	8d9e0153be8bfe0c936ac4e30cbcbecd.bam	Blood Derived Normal	WGS	FF1500F7-8FC1-4ACD-B93D-375BF881B90A
```

Comment:
I don't think we have info for BarcodeSequence, LinkerPrimerSequence, you can just leave them blank (as a tab).